efficientdet/main.py

```diff
@@ -347,6 +357,13 @@ def _eval(steps):
       eval_results = _eval(eval_steps)
       ckpt = tf.train.latest_checkpoint(FLAGS.model_dir)
       utils.archive_ckpt(eval_results, eval_results['AP'], ckpt)
+    for e in range(current_epoch + 1, config.num_epochs + 1, epochs_per_cycle):
+      if FLAGS.each_epoch_in_separate_process:
+        p = multiprocessing.Process(target=lambda: run_train_and_eval(e))
```

Comment:
Thank you! This is a brilliant idea.  Does it solve the memory leak issue?

Reply:
Yes, it does

Reply:
i think you need to terminate process explicitly.

Reply:
Why do you want to terminate rather than wait for the process to finish? I can add p.close() before p.join(). I tested it and it is working. But I'm not sure what is the use for p.close() in here. Is it something you are looking for?

Reply:
nah we cool , you already explained it below.